src/ploneorg/releasesecurityinfo/models/release.xml

```diff
@@ -52,15 +49,44 @@
         <required>False</required>
       </field>
 
-      <field name="description"
+      <field name="summary"
              type="zope.schema.Text"
              security:write_permission="cmf.ManagePortal"
              security:read_permission="zope2.View">
-        <title i18n:translate="">Description</title>
+        <title i18n:translate="">A summary of the features and status of this milestone.</title>
+        <description i18n:translate=""></description>
+        <required>False</required>
+      </field>
+
+      <field name="series"
```

Comment:
I would remove this field, this is implicit given by the ReleaseSeries Folder this Release is added to